app/views/catarse_bootstrap/projects/index.html.slim

```diff
@@ -1,7 +1,7 @@
 - content_for :title, t('site.title')
 = render 'shared/header_big', overlayer: true
 
-#project-index-root[data-mithril="ProjectsHome"]
+#project-index-root[data-mithril="ProjectsHome" data-currentuserid=current_user.try(:id) data-hasfb="#{current_user.try(:has_fb_auth?)}"]
```

Comment:
What do you think of adding those two fields to the data-user attribute that is already on the body tag?